packages/wouter-preact/test/preact.test.tsx

```diff
@@ -33,15 +33,15 @@ describe("Preact support", () => {
 
         <main data-testid="routes">
           <Switch>
-            Welcome to the list of {100} greatest albums of all time!
+            <>Welcome to the list of {100} greatest albums of all time!</>
             <Route path="/albums/all">Rolling Stones Best 100 Albums</Route>
-            <Route path="/albums/:name">
+            <Route<{ name: string }> path="/albums/:name">
```

Comment:
Hmm, routes should infer the proper type of parameters from the path.

Reply:
I managed to fix this